### ISSUE TITLE
approval-voting: enable v2 assignments

### ISSUE THREAD

**Issue by sandreim** [paritytech/polkadot#7269]
Once https://github.com/paritytech/polkadot/pull/6782 is merged both v1/v2 will be supported. The plan is to enable v2 assignment generation once enough validators have upgraded but only for Kusama and the other testnets. Once we rule out any issues we should also enable on Polkadot.

**Comment by burdges** [comment]
Can we hit testnets before Kusama?  I've one thing it'd be nice to squeeze into this.

**Comment by sandreim** [comment]
Yes, we will enable only after testnets. What are you thinking to squeeze in?
